fleet-be/src/test/java/com/everis/training/fleet/business/fleet/control/VehicleControllerTest.java

```diff
@@ -17,12 +17,6 @@
     @Mock
```

Comment:
In the class VehicleControllerTest the are some duplicated lines of code to validate the vin, there is some problems with that:
- Duplicated line of code, harder to be maintained.
- Null check is missing, if vin = null it would throw a NPE.
- The caller to the method is not aware that the action (insert, update...) was not perfomed due to a business requirement. An exception should be thrown.